etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -29,6 +29,7 @@ var (
 	ErrGRPCDuplicateKey  = grpc.Errorf(codes.InvalidArgument, "etcdserver: duplicate key given in txn request")
 	ErrGRPCCompacted     = grpc.Errorf(codes.OutOfRange, "etcdserver: mvcc: required revision has been compacted")
 	ErrGRPCFutureRev     = grpc.Errorf(codes.OutOfRange, "etcdserver: mvcc: required revision is a future revision")
+	ErrGRPCPastRev       = grpc.Errorf(codes.FailedPrecondition, "etcdserver: required response revision to be higher than previous revisions")
```

Comment:
please remove these errors, this file is only meant for errors send out by the etcd server over grpc (and nothing in this patch uses the errors anyway)